### PR TITLE
upgrade validator version

### PR DESCRIPTION
**Pull request by chunhui2001**

### CODE REVIEW

**Comment by DenuxPlays** [comment]
Tested it.
Works great!

File: Cargo.toml [hunk 1]
```diff
@@ -1,20 +1,26 @@
 [package]
-name = "actix-web-validator"
-version = "5.0.1"
-authors = ["Belousow Makc <lib.bmw@gmail.com>", "Andrey Ermilov <andrerm@ya.ru>", "Kolomatskiy Artem <kolomatskiy@bk.ru>"]
+name = "actix-web-validator5"
+version = "1.0.0"
+authors = ["keesh.zhang<chunhui2001@gmail.com>"]
```
Comment (DasKoebi) [inline Cargo.toml:4]: Was this change intended?

File: Cargo.toml [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+    "validator",
+    "validation",
+]
 categories = ["encoding", "web-programming"]
 license = "MIT"
 readme = "README.md"
-repository = "https://github.com/rambler-digital-solutions/actix-web-validator"
-documentation = "https://docs.rs/actix-web-validator/"
+repository = "https://github.com/chunhui2001/actix-web-validator"
+documentation = "https://docs.rs/actix-web-validator5"
```
Comment (DasKoebi) [inline Cargo.toml:18]: Was this change intended?

**Review by DasKoebi — CHANGES_REQUESTED** [review]
I dont think this PR will be merged with the following changes.

**Comment by singulared** [comment]
Thanks a lot for your work but i close this PR in prefer to #55

**Pull request closed without merging**